Blish HUD/GameServices/Input/Keyboard/TypedInputUtil.cs

```diff
@@ -14,142 +9,86 @@ internal static class TypedInputUtil {
         // because of this behavior, "^" is called dead key)
 
         [DllImport("user32.dll")]
-        private static extern int ToUnicodeEx(uint wVirtKey, uint wScanCode, byte[] lpKeyState, [Out, MarshalAs(UnmanagedType.LPWStr)] System.Text.StringBuilder pwszBuff, int cchBuff, uint wFlags, IntPtr dwhkl);
+        private static extern int ToUnicode(uint vkCode, uint scanCode, byte[] keyState, [Out, MarshalAs(UnmanagedType.LPWStr, SizeConst = 64)] System.Text.StringBuilder receivingBuffer, int bufferSize, uint flags);
```

Comment:
I might have missed it.  Was there any reason in particular we decided to go for `ToUnicode` instead of `ToUnicodeEx` for this?

Reply:
(this is not a blocker)

Reply:
It may have had something todo with letting the OS handle the keyboard layout and not ourselfs since ``ToUnicodeEx`` expects a local identifier used to translate the code ie. it appears to be just a means to force a different layout for translation if you'd want that. ``ToUnicode`` uses the current one.

Reply:
Yes. @greaka and I were discussing this. Since we don't need to inject our own layout at any point we ditched the `ToUnicodeEx`. Reading the current layout and injecting it would have been unnecessary and redundant code.

Reply:
`ToUnicodeEx` requires us to handle keyboard layouts ourselves which `ToUnicode` does on its own. When a user switches layouts, we either have to poll layouts again every update or we don't support switching layouts. We just don't want or need the control `ToUnicodeEx` provides

Reply:
Alright, thanks for the explanations!  That makes perfect sense.